test/github.test.ts

```diff
@@ -360,6 +360,30 @@ test('example app', () => {
   });
 });
 
+test('pipeline with concurrency', () => {
+  withTemporaryDirectory((dir) => {
+    const github = new GitHubWorkflow(app, 'Pipeline', {
+      workflowPath: `${dir}/.github/workflows/deploy.yml`,
+      synth: new ShellStep('Build', {
+        installCommands: ['yarn'],
+        commands: ['yarn build'],
+      }),
+      concurrency: {
+        group: '${{ github.workflow }}-${{ github.ref }}',
+      },
+    });
+
+    app.synth();
+
+    console.log(github.workflowPath);
+    console.log(github.workflowPath);
+    console.log(github.workflowPath);
+    console.log(github.workflowPath);
```

Comment:
Is this part of the concurrency test? how? please document otherwise next time i look at this i will laugh and then delete...